### DISCUSSION TITLE
测试AGIEval数据集时，模型生成结果output中的prediction文件夹和result文件夹中的文件数量不能完全对应

### DISCUSSION THREAD

**Discussion by 123456xxn** [open-compass/opencompass#900] [Q&A]
![agi_result](https://github.com/open-compass/opencompass/assets/128134019/feb0d655-d22b-43f6-ab8e-f87182ee2887)

请问result中报告的结果是否是相近文件名的总分数？比如：agieval-jec-qa-ca.json中的结果score是agieval-jec-qa-ca_0.json，agieval-jec-qa-ca_1.json，agieval-jec-qa-ca_2.json，agieval-jec-qa-ca_3.json，agieval-jec-qa-ca_4.json的总结果吗？

**Comment by Leymore** [comment]
The `_0.json`, `_1.json`, etc., in the prediction folder are the outputs of the partitioner; the result will merge the partitioned predictions to obtain the complete score.
